wgpu-core/src/lib.rs

```diff
@@ -234,14 +234,12 @@ macro_rules! gfx_select {
             wgt::Backend::Vulkan => $global.$method::<$crate::api::Vulkan>( $($param),* ),
             #[cfg(all(not(target_arch = "wasm32"), any(target_os = "ios", target_os = "macos")))]
             wgt::Backend::Metal => $global.$method::<$crate::api::Metal>( $($param),* ),
-            /*
-            #[cfg(all(not(target_arch = "wasm32"), windows))]
-            wgt::Backend::Dx12 => $global.$method::<$crate::backend::Dx12>( $($param),* ),
-            #[cfg(all(not(target_arch = "wasm32"), windows))]
-            wgt::Backend::Dx11 => $global.$method::<$crate::backend::Dx11>( $($param),* ),
-            #[cfg(any(target_arch = "wasm32", all(unix, not(any(target_os = "ios", target_os = "macos")))))]
-            wgt::Backend::Gl => $global.$method::<$crate::backend::Gl>( $($param),+ ),
-            */
+            //#[cfg(all(not(target_arch = "wasm32"), windows))]
+            //wgt::Backend::Dx12 => $global.$method::<$crate::api::Dx12>( $($param),* ),
+            //#[cfg(all(not(target_arch = "wasm32"), windows))]
+            //wgt::Backend::Dx11 => $global.$method::<$crate::api::Dx11>( $($param),* ),
+            #[cfg(all(not(target_arch = "wasm32"), unix, not(any(target_os = "ios", target_os = "macos"))))]
```

Comment:
You planned to implement WebGL 2 as separate target ? Or `not(target_arch = "wasm32")` because WebGL2 still not ready?

Reply:
Just not ready yet.